packages/core/src/constants/index.ts

```diff
@@ -10,8 +10,14 @@ export {
   defaultChains,
   defaultL2Chains,
 } from './chains'
+export type { ChainName } from './chains'
```

Comment:
Too bad ethers network names have dashes in them, otherwise `ChainName` could of been the same.